catena4450m101_sensor/catena4450m101_sensor.ino

```diff
@@ -539,7 +539,8 @@ static void settleDoneCb(
 
     // if connected to USB, don't sleep
     // ditto if we're monitoring pulses.
-    if (Serial.dtr() || fHasPower1)
```

Comment:
I don't think we should change the sketch by default. Perhaps this crept in by mistake?